pkg/sdkserver/sdkserver_test.go

```diff
@@ -718,7 +718,7 @@ func TestSDKServerWatchGameServerFeatureSDKWatchSendOnExecute(t *testing.T) {
 			} else {
 				assert.Fail(t, "Channel is closed!")
 			}
-		default:
+		case <-time.After(30 * time.Second):
```

Comment:
Does this still need to try 2 times? If you are going to add a timeout, maybe remove the for statement?

Reply:
This test is looking for two events to be passed through - the initial one on connection, then the second one from WatchGameServer.

I could rewrite it as two `select` statements, but that feels a little verbose?

Reply:
I didn't catch that part. I think it might be a bit clearer and/or more idiomatic to do a `for { }` loop with a break (when it times out or when you've gotten the expected events) but this is ok too.

Reply:
Would there be a concern that it would never escape the loop though?

Reply:
You would escape the loop as long as you add break / return (or panic) statements to each case, e.g.

```
for {
  select {
  case _, ok := <-stream.msgs:
    // do stuff - maybe assert that the right messages show up?
    break
  case <-time.After(timeout):
    // error, since we hit a timeout when we were expecting messages
    t.Fail() // should cause the test to end, so no need to break / return
  }
}
```